PyDynamic/misc/tools.py

```diff
@@ -31,8 +31,45 @@
     "make_equidistant",
     "trimOrPad",
     "progress_bar",
+    "shift_uncertainty"
 ]
 
+def shift_uncertainty(x, ux, shift):
+    """Shift the elements in the vector x (and associated uncertainty ux) by shift elements
+
```

Comment:
Amend docstring with a remark that np.roll is used for shifting.
```suggestion
    """Shift the elements in the vector x (and associated uncertainty ux) by shift elements.
        This method uses :class:`numpy.roll` to shift the elements in x and ux. See documentation of np.roll for details.

```